cgi-bin/DW/Logic/ProfilePage.pm

```diff
@@ -838,6 +838,17 @@ sub external_services {
         };
     }
 
+    if ( my $emedium = $u->prop( 'medium' ) ) {
+	my $emedium = LJ::eurl( $emedium );
+	push @ret, {
+	    type => 'medium',
+	    text => LJ::ehtml( $emedium ),
```

Comment:
You don't want to double-escape something -- in this case you eurl'd the contents and then you're ehtml'ing the eurl'd comments which won't work out well.